atm/Sidorchuk/task15/ATM/ATM.java

```diff
@@ -0,0 +1,101 @@
+package task15.ATM;
+
+import task15.ATM.money.Currency;
+
+public class ATM {
+
+    private long totalAmount = 1000000;
+    private Card card;
+    ATMSecurity atmSecurity = new ATMSecurity();
```

Comment:
Better to create in a main method and inject it into the constructor since it can have multiple implementations